### PR TITLE
fix and small balance change to the utility vendor.

### PR DESCRIPTION
**Pull request by Huffie56**
# About the pull request
this is to first fix a single mistake of duplicate item being sell : flashlight.
also add multi-tool roller bed and mini extinguisher.

# Explain why it's good for the game
fixing and balancing in small touch is good.
# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
add: Add roller bed, mini extinguisher and multi-tool to the utility vendor in small quantity.
balance: divide by two the number of "ME3 hand welder" in utility vendor.
fix: remove a duplicate item being sell the flashlight.
fix: some clothing items have a minimum of one now.
/:cl:

### CODE REVIEW

**Comment by Huffie56** [comment]
need to add jungle boots and USCM and falcon patch to the clothing vendor.

File: code/game/machinery/vending/vendor_types/squad_prep/squad_prep.dm [hunk 1]
```diff
@@ -71,6 +71,7 @@
 	listed_products = list(
 		list("STANDARD EQUIPMENT", -1, null, null, null),
 		list("Marine Combat Boots", round(scale * 15), /obj/item/clothing/shoes/marine, VENDOR_ITEM_REGULAR),
+		list("Jungle Pattern Combat Boots", round(scale * 1), /obj/item/clothing/shoes/marine/jungle, VENDOR_ITEM_REGULAR),
```
Comment (morrowwolf) [inline code/game/machinery/vending/vendor_types/squad_prep/squad_prep.dm:74]: These need to stay as buyable accessories, not standard issue

**Review by morrowwolf — CHANGES_REQUESTED** [review]
Otherwise fine

File: code/game/machinery/vending/vendor_types/squad_prep/squad_prep.dm (hunk 1 above)
Comment (Huffie56) [reply code/game/machinery/vending/vendor_types/squad_prep/squad_prep.dm]: you might want to take a look at miscelanous in the clothing vendor line 149 then...

File: code/game/machinery/vending/vendor_types/squad_prep/squad_prep.dm [hunk 2]
```diff
@@ -272,20 +275,24 @@
...
 
 		list("FLARE AND LIGHT", -1, null, null),
 		list("Combat Flashlight", round(scale * 5), /obj/item/device/flashlight/combat, VENDOR_ITEM_REGULAR),
-		list("Flashlight", round(scale * 5), /obj/item/device/flashlight/combat, VENDOR_ITEM_REGULAR),
 		list("Box of Flashlight", round(scale * 1), /obj/item/ammo_box/magazine/misc/flashlight, VENDOR_ITEM_REGULAR),
 		list("Box of Flares", round(scale * 1), /obj/item/ammo_box/magazine/misc/flares, VENDOR_ITEM_REGULAR),
 		list("M94 Marking Flare Pack", round(scale * 10), /obj/item/storage/box/m94, VENDOR_ITEM_REGULAR),
 		list("M89-S Signal Flare Pack", round(scale * 1), /obj/item/storage/box/m94/signal, VENDOR_ITEM_REGULAR),
+		list("Box of Blue Lightsticks", round(scale * 1), /obj/item/storage/box/lightstick, VENDOR_ITEM_REGULAR),
+		list("Box of Red Lightsticks", round(scale * 1), /obj/item/storage/box/lightstick/red, VENDOR_ITEM_REGULAR),
```
[the remark this answers is not in the document]
Comment (Huffie56) [reply code/game/machinery/vending/vendor_types/squad_prep/squad_prep.dm]: copy removed

**Review by morrowwolf — APPROVED** [review]

**Pull request merged**